src/tiles/grid/tile-stats.ts

```diff
@@ -43,7 +43,7 @@ export class TileStats extends LitElement {
             <p class="sr-only">Mediatype:</p>
             <mediatype-icon .mediatype=${this.mediatype}></mediatype-icon>
           </li>
-          <li class="col" title="${uploadsOrViewsTitle}">
+          <li class="col views" title="${uploadsOrViewsTitle}">
```

Comment:
views and favorites class is missing here, only reviews is declared in the CSS styles.
can we come up with a more generic class name for this instead?

Reply:
We can certainly remove the unused "views" and "favorites" classes (unless we need to make different adjustments to those icons too).

For the reviews icon though, I am _specifically_ targeting that one for a tailored adjustment that likely doesn't generalize elsewhere...